services/vfs_service/assembler.go

```diff
@@ -71,7 +71,7 @@ const (
     "sort": [
       {
         "timestamp": {
-          "order": "asc"
```

Comment:
I am not 100% sure about this - the asc here is used to ensure that if we download a file then download it again the second download will override the first - can you please make sure this is still the case?

Reply:
@scudette asc is the earliest record, desc is the latest record since it is an epoch value